prusti-common/src/vir/ast/expr.rs

```diff
@@ -995,6 +1019,9 @@ impl Expr {
             Expr::DomainFuncApp(ref func, _, _) => {
                 &func.return_type
             },
+            Expr::BackendFuncApp(ref func, _, _) => { 
+                unimplemented!("Unsure what comes here")
```

Comment:
@vakaras See unimplemented! message

Reply:
This should return the return type of the function. I guess it should be similar to `DomainFuncApp`.